proposals/description/fip_72.ts

```diff
@@ -0,0 +1,94 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_73: ProposalDescription = {
+  title: 'FIP-72: New DAI PSM',
+  commands: [
+    {
+      target: 'core',
+      values: '0',
+      method: 'grantMinter(address)',
+      arguments: ['{daiFixedPricePSM}'],
+      description: 'Grant DAI psm minter role'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokeMinter(address)',
+      arguments: ['{daiPSM}'],
+      description: 'Remove minter role from old DAI psm'
+    },
+    {
+      target: 'core',
+      values: '0',
+      method: 'revokePCVController(address)',
+      arguments: ['{daiPSMFeiSkimmer}'],
+      description: 'Revoke daiPSMFeiSkimmer PCV controller role from old DAI PSM FEI skimmer'
```

Comment:
should be removed if new skimmer not being added in this pr

Reply:
no, this should be removed because it's the old PSM dai PSM skimmer and that old dai psm is getting deprecated with this PR